### PR TITLE
build: add aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by lgray**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Comment by lgray** [comment]
@matthewfeickert it seems to only be failing for some combinations, please have a look!

**Comment by matthewfeickert** [comment]
Oh man. Travis CI error logs. A blast from the past that's giving me some mental and emotional damage. :grimacing: 

The build matrix failures are confusing too as they seem to only be CPython version based. :?

image

I'll have to dig through and also learn what

```
"if [[ ${PLATFORM} =~ .*linux.* ]]; then CONDA_FORGE_DOCKER_RUN_ARGS="--network=host --security-opt=seccomp=unconfined" ./.scripts/run_docker_build.sh; fi" exited with 137.
```

is all about.

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Comment by lgray** [comment]
@matthewfeickert may have found an easier way through - just use azure instead ^_^

**Comment by lgray** [comment]
While this might take a while since it's emulated, I imagine it is far better than trying to setup this build system for cross compilation.

**Comment by matthewfeickert** [comment]
> While this might take a while since it's emulated, I imagine it is far better than trying to setup this build system for cross compilation.

@lgray yeah if we can avoid using Travis CI I would be very happy. I assume there's no way to use [Cirrus CI](https://cirrus-ci.org/) as a provider? If not, as long as this stays under 2 hours of build time I think that's okay (we can also turn things off as needed if we need to debug in a PR).

File: conda-forge.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```
Comment (matthewfeickert) [inline conda-forge.yml:9]: @lgray can you link to the docs that you read that cover providers? I'm trying to assemeble some conda-forge bookmarks for myself for the future. :grimacing:

**Comment by lgray** [comment]
Huh, the builds in azure keep getting to:
```
libtool: compile:  $BUILD_PREFIX/bin/aarch64-conda-linux-gnu-c++ -DHAVE_CONFIG_H -I. -I../include/fastjet -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem $PREFIX/include -g -Woverloaded-virtual -frounding-math -I./../include -fvisibility-inlines-hidden -fmessage-length=0 -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -pipe -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/fastjet-3.4.0.6 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -O3 -Bstatic -Bdynamic -std=c++17 -MT libfastjet_la-RangeDefinition.lo -MD -MP -MF .deps/libfastjet_la-RangeDefinition.Tpo -c RangeDefinition.cc -o libfastjet_la-RangeDefinition.o >/dev/null 2>&1
```
and then completing wreck the node they're on for some reason.

File: conda-forge.yml (hunk 1 above)
Comment (lgray) [reply conda-forge.yml:9]: https://conda-forge.org/docs/maintainer/conda_forge_yml/#provider

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Comment by lgray** [comment]
@conda-forge-admin, please rerender

**Comment by matthewfeickert** [comment]
Could also try using Circle CI: https://circleci.com/docs/using-arm/

But it is strange that it keeps failing at that one spot.

**Comment by lgray** [comment]
@conda-forge-admin, please update circle.

**Comment by matthewfeickert** [comment]
@lgray have you actually succeeded in building the `fastjet` wheel on aarch_64 Linux machine before? If not, I can try to build on an RPi and see where things break down.

**Comment by lgray** [comment]
Oh it works on cirrus no problem and we use it to build wheels quickly. The builds were completing on travis in some cases just fine as well. It really seems to be a worker memory issue in some versions of python.

ppc64le was also succeeding for certainly python versions, just not all of them.

**Comment by lgray** [comment]
@conda-forge-admin, please add circle

**Comment by matthewfeickert** [comment]
> Oh it works on cirrus no problem and we use it to build wheels quickly

Oh right. I forgot that I added the PR for that. https://github.com/scikit-hep/fastjet/pull/168 :zany_face: Time to step away from the keyboard and make a coffee.

Thanks for steering this and getting these builds going in the right direction.

**Comment by lgray** [comment]
@chrisburr does conda-forge no longer actually use circle-ci or do I need to go through some extra hoops I don't know about?

**Comment by lgray** [comment]
I have a feeling I need to first merge this and then it'll work.

**Pull request merged**